host.md

```diff
@@ -1,36 +1,138 @@
+- There must be amplification of in-room speakers when they use the microphone.
+- The host is encouraged to provide video conferencing using Zoom or Google Meet.
+  - Ideally the room has at least one rear-facing camera near the front that can provide a view of all the participants.
+  - We encourage a tech-check to be held on the Monday immediately prior to the meeting to verify the full AV setup works.
+- Whiteboards and marker pens are optional.
+
+### Food & Drink
+
+- Tea and coffee should be available throughout the day.
+- Breakfast and Lunch must be provided for all attendees.
+- Breakfast is usually from 09:00.
+  - Most people will arrive around 09:30.
+  - Food does not need to be cleared when the meeting starts - it is fine to leave food out for latecomers.
+- Lunch is usually at 12:00.
+  - The lunch location should be on-site to minimize any travel time.
+  - It is important this is held in an area that is reasonably exclusive to permit delegates to easily talk in smaller groups.
+  - Shared cafeterias are discouraged unless a designated area can be secured.
+- Vegan/vegetarian options must be available.
+- Individual's dietary requirements are collected on the registration form.
+  - Chairs will relay this information to the host two weeks before the meeting.
+
+### Social Dinner
+
+- On one evening the host will arrange a social dinner at a nearby restaurant.
+  - It is usually Wednesday but could be Tuesday.
+- Make sure you reserve the restaurant well ahead of time.  It is hard to get a room for 30+ people so getting started early is recommended.
+- Ecma will share half of the cost.
+  - Please liase with [the TC39 Secretary](samina@ecma-international.org) to discuss this.
+
+### Community Event
+
+- We try to meet with the local JavaScript community on one evening.
+  - It is usually Thursday but other days, including Monday, are possible.
+- Hosts are free to create their own event, but usually it is easiest for them to seek out a local Meetup group who have a pre-existing list of members.
+  - You are welcome to host it at the same location as the meeting or to have it somewhere reasonably close, e.g. <30 mins travel time.
+- TC39 Delegates are usually happy to give talks.
+- Often we end with a Q&A panel taking questions from a moderator or the audience.
+- Feel free to be creative.  Delegates also enjoy hearing talks from the community.
+- Recording/streaming of the event is possible so long as that is communicated in advance and individuals who do not wish to be captured can opt-out.
+
+## Timeline of Events
+
+This is the general timeline of events leading up to the meeting.
+
+1. Reach out to the chairs for early discussions.
+1. Book the meeting room and confirm you have budget secured.
+1. The chairs will announce the meeting, normally as part of the annual schedule.
+1. 3-4 months before the meeting... the Chairs will conduct an Interest Survey to determine go/no-go.
```

Comment:
Why `...` in this sentence and in the next ones?

Reply:
It was meant to just be a recurring visual separation of date from activity. Happy to delete if you think it's not adding value.